test/build/metadata-dir/index.test.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs');
+const path = require('path');
+const execa = require('execa');
+
+it('buildOptions.metadataDir', () => {
```

Comment:
fine for our first test, but our 2nd/3rd test should move this into a runner similar to `test/integrations` so that we can standardize the test structure.